addons/l10n_sk/data/account_fiscal_position_data.xml

```diff
@@ -68,5 +80,17 @@
             <field name="tax_dest_id" ref="vs_nad_eu" />
         </record>
 
+        <record id="fiscal_position_tax_template_21c" model="account.fiscal.position.tax.template">
+            <field name="position_id" ref="fiscal_position_template_2"  />
+            <field name="tax_src_id" ref="vs_tuz_23" />
+            <field name="tax_dest_id" ref="vs_nad_eu" />
+        </record>
+
+        <record id="fiscal_position_tax_template_21d" model="account.fiscal.position.tax.template">
```

Comment:
It is so wrong btw :smile: (the 19% is covered by a 23% one)
Not your fault, as it's the same as before but this localization definitely needs to have a re-pass
Between that, the fact we have no tax report and that the fiscal position are in the wrong order, there is some work to do here in a later task